artemis-cli/src/main/java/org/apache/activemq/artemis/util/ServerUtil.java

```diff
@@ -32,6 +32,7 @@
 
 /**
  * A tool to let clients start, stop and kill Artemis servers
+ * this class is Deprecated: use the one from artemis-test-support
```

Comment:
I am creating RealServerTestBase as a parent class, so it can be reused between smoke-tests, db-tests, soak-tests (and future ones). RealServerTestBase comes before the CLI.

There's one example that's using ServerUtil.   And ServerUtil is really meant for testing only.

Reply:
Actually.. there are more examples... I'm adding tests-support on these examples just to see how it looks...

Reply:
You cant currently do that, we dont publish test-support as its only meant for use in the tests subtree.

Reply:
solved! Thank you.